landside/catkin_ws/src/simulation/scripts/sim_loop.py

```diff
@@ -65,15 +64,18 @@ def publish_depth(self, pose, twist):
 
     def run(self):
         rate = rospy.Rate(10)  # 10 Hz
+        num_cycles = 0
         while not rospy.is_shutdown():
-            pose = self.sim_handle.get_pose()
+            pose = self.sim_handle.get_pose(mode=sim.simx_opmode_blocking)
```

Comment:
I believe I removed it because it returns an error code the first time it runs because it doesn't get a return value from the simulation. I will double check

Reply:
Here's the exact console output if I set it to streaming instead of blocking:
```
[ERROR] [1645146676.580118]: sim_handle.run_sim_function: Error calling simulation. Code: 1
[ERROR] [1645146676.584108]: sim_handle.run_sim_function: Function: <function simxGetObjectPosition at 0x7f610f9a2940>
[ERROR] [1645146676.604210]: sim_handle.run_sim_function: Args: (0, 13, -1, 393216)
  File "/root/dev/robosub-ros/landside/catkin_ws/src/simulation/scripts/sim_loop.py", line 84, in <module>
    SimLoop().run()
  File "/root/dev/robosub-ros/landside/catkin_ws/src/simulation/scripts/sim_loop.py", line 69, in run
    pose = self.sim_handle.get_pose()
  File "/root/dev/robosub-ros/landside/catkin_ws/src/simulation/scripts/sim_handle.py", line 93, in get_pose
    pos = self.run_sim_function(sim.simxGetObjectPosition, (self.clientID, self.robot, -1, mode))
  File "/root/dev/robosub-ros/landside/catkin_ws/src/simulation/scripts/sim_handle.py", line 61, in run_sim_function
    traceback.print_stack()
```